src/parser/Expression.ts

```diff
@@ -1476,8 +1477,9 @@ export class NullCoalescingExpression extends Expression {
 
     transpile(state: BrsTranspileState) {
         let result = [] as TranspileResult;
-        let consequentInfo = util.getExpressionInfo(this.consequent);
-        let alternateInfo = util.getExpressionInfo(this.alternate);
+        const file = state.file;
+        let consequentInfo = util.getExpressionInfo(this.consequent, file);
+        let alternateInfo = util.getExpressionInfo(this.alternate, file);
```

Comment:
No need to make a separate variable here, just reference it from state.

```suggestion
        let consequentInfo = util.getExpressionInfo(this.consequent, state.file);
        let alternateInfo = util.getExpressionInfo(this.alternate, state.file);
```